src/encoded/types/acl.py

```diff
@@ -102,12 +109,14 @@
 
 # This gives item owners expanded permissions
 ALLOW_OWNER_EDIT_ACL: Acl = [
-    (Allow, 'role.owner', ['edit', 'view', 'view_details']),
+    (Allow, OWNER_ROLE, ['edit', 'view', 'view_details']),
 ] + ONLY_ADMIN_VIEW_ACL
 
 # These two ACLs allow item editing
 SUBMISSION_CENTER_MEMBER_EDIT_ACL: Acl = [
-    (Allow, SUBMISSION_CENTER_RW, ['view', 'edit'])
+    # Note that only submitters can edit
+    (Allow, SUBMITTER_ROLE, ['edit']),
```

Comment:
Should this include `view` as well? I believe a User can now upload things for a center to which they don't belong. May be useful to include some kind of validation on User to prevent this if it's an undesirable state.